src/main/java/org/opentripplanner/routing/api/request/request/TransitRequest.java

```diff
@@ -4,24 +4,20 @@
 import java.util.ArrayList;
 import java.util.List;
 import org.opentripplanner.routing.api.request.DebugRaptor;
-import org.opentripplanner.routing.core.RouteMatcher;
-import org.opentripplanner.transit.model.basic.MainAndSubMode;
+import org.opentripplanner.routing.api.request.request.filter.TransitFilter;
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 
 // TODO VIA: Javadoc
 public class TransitRequest implements Cloneable, Serializable {
 
-  private List<MainAndSubMode> modes = MainAndSubMode.all();
+  private List<FeedScopedId> bannedTrips = new ArrayList<>();
```

Comment:
Should these not be converted to a filter?

Reply:
Nope. This is not included in new filter API since having multiple list for banned trips is pointless. It should stay here.